src/main/java/roboy/dialog/DialogSystem.java

```diff
@@ -80,19 +76,21 @@ public static void main(String[] args) throws IOException {
         //analyzers.add(new OntologyNERAnalyzer());
         analyzers.add(new AnswerAnalyzer());
 
+        InferenceEngine inference = new Inference();
 
         logger.info("Creating StateBasedPersonality...");
 
-        StateBasedPersonality personality = new StateBasedPersonality(rosMainNode, memory, new Verbalizer());
+        StateBasedPersonality personality = new StateBasedPersonality(rosMainNode, memory, inference, new Verbalizer());
         File personalityFile = new File(ConfigManager.PERSONALITY_FILE);
 
 
 
         // Repeat conversation a few times
-        for (int numConversations = 0; numConversations < 3; numConversations++) {
+        for (int numConversations = 0; numConversations < 50; numConversations++) {
 
             logger.info("############## New Conversation ##############");
-
+            logger.info("PRESS ENTER TO START THE DIALOG. ROBOY WILL WAIT FOR SOMEONE TO GREET HIM (HI, HELLO)");
+            System.in.read();
```

Comment:
this worked well for the expo but has to be removed now

Reply:
would disagree, cause we are spending precious free calls to the speech recognition and very rarely we have a sequence of conversation without a pause in between. for debugging doesnt make a difference

Reply:
hmm, you are right about calls to speech recognition
let's keep it